exchange/utils.go

```diff
@@ -117,3 +119,17 @@ func prepareUser(req *openrtb.BidRequest, bidder openrtb_ext.BidderName, usersyn
 		}
 	}
 }
+
+func makeLoggableAdapterRequests(name openrtb_ext.BidderName, reqData []*adapters.RequestData) []analytics.LoggableAdapterRequests {
+	ar := make([]analytics.LoggableAdapterRequests, len(reqData))
+	for i, req := range reqData {
+		ar[i] = analytics.LoggableAdapterRequests{
+			Name:     string(name),
+			Requests: string(req.Body),
```

Comment:
These names are a little confusing... a few things are plural when I think you mean for them to be singular.

Maybe rename `LoggableAdapterRequests` to `LogableAdapterRequest`, and `Requests` to `RequestBody`?